Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Bugzilla44461.cs

```diff
@@ -1,19 +1,36 @@
-﻿using Xamarin.Forms.CustomAttributes;
+﻿using System;
+using System.Linq;
+using Xamarin.Forms.CustomAttributes;
 using Xamarin.Forms.Internals;
 
+#if UITEST
+using Xamarin.UITest;
+using Xamarin.UITest.Queries;
+using NUnit.Framework;
+#endif
+
 namespace Xamarin.Forms.Controls
```

Comment:
```suggestion
namespace Xamarin.Forms.Controls.Issues
```

The UI Test are failing because if this namespace isn't *Xamarin.Forms.Controls.Issues* then the startup code doesn't run for NUnit